config/initializers/secure_headers.rb

```diff
@@ -13,9 +13,14 @@
     :enforce     => true,
     :default_src => 'self',
     :frame_src   => 'self',
-    :connect_src => 'self ws: wss:',
+    :connect_src => %w(self ws: wss:),
     :style_src   => 'inline self',
-    :script_src  => 'eval inline self',
-    :img_src     => ['self', '*.gravatar.com']
+    :script_src  => %w(eval inline self),
+    :img_src     => %w(self *.gravatar.com)
   }
+  if Rails.env.development? #allow webpack dev server provided assets
+    dev_server = "http://0.0.0.0:#{::Rails.configuration.webpack.dev_server.port}"
```

Comment:
the default Rails.configuration.webpack.dev_server.manifest_host is localhost afaiu, hence it will not work unless you change it to 0.0.0.0